includes/mutation/class-cart-remove-coupons.php

```diff
@@ -85,6 +85,9 @@ public static function mutate_and_get_payload() {
 				}
 			}
 
+			// Recalculate totals.
+			\WC()->cart->calculate_totals();
```

Comment:
I think this is a much deeper problem. Cart total recalculation is actually done by WooCommerce during the `WC()->cart->remove_coupon()`.

Reply:
🤔Okay. Are you referring to refresh totals? https://woocommerce.wp-a2z.org/oik_api/wc_cartremove_coupon/

Reply:
Yes. This issue could also be an issue how the `cart` is being resolved as well.

Reply:
@zamson can you check and see if this issue still persist on [Release v0.5.1](https://github.com/wp-graphql/wp-graphql-woocommerce/releases/tag/v0.5.1)

Reply:
Yes it's still there.